.github/workflows/ci.yaml

```diff
@@ -56,22 +56,40 @@ jobs:
       - name: Build package
         run: poetry build
         working-directory: agent
-      - name: Publish package
-        uses: actions/upload-artifact@v4
-        with:
-          name: ${{ github.event_name == 'pull_request' && format('dist-pr-{0}', github.event.number) || 'dist' }}
-          path: agent/dist/*
-          retention-days: 5
       - name: Get version from pyproject.toml
         id: get_version
         run: |
           VERSION=$(sed -n '3p' agent/pyproject.toml | cut -d'"' -f2)
           echo "version=$VERSION" >> "$GITHUB_OUTPUT"
+      - name: Set up Node.js for browser service
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+      - name: Build browser service
+        run: |
+          cd browser
+          npm install
+          npm run build
+          cd ..
```

Comment:
We should fix this up:
- remove the `cd` statements, the package json is in the root of the repo
- Switch the `npm run build` command to `browser:build`
- define a `prebrowser:build` script in the package.json that just `npm install` so that this step is one command